package.json

```diff
@@ -42,7 +42,7 @@
 		"redux-thunk": "~2.3.0",
 		"styled-components": "5.0.1",
 		"styled-system": "5.1.2",
-		"synthetix-data": "2.1.23",
+		"synthetix-data": "^2.1.34",
```

Comment:
do we wanna remove the ^ for the packages as we've done on the other dependencies

Reply:
yeah right, will do!